openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -579,6 +584,11 @@ def _validate_password(password, username=None, email=None):
         raise errors.AccountPasswordInvalid(' '.join(validation_err.messages))
 
 
+def _validate_confirm_password(confirm_password, password):
+    if not confirm_password or confirm_password != password:
```

Comment:
This check `not confirm_password` should have a separate error message. Or if you required the field in another step, you can ignore this condition.

Reply:
This part validates the confirm password field if the user moves their cursor outside of the box. Updated in latest commit to have a more informative message when the input is blank